api/src/main/java/com/codeforcommunity/rest/ApiRouter.java

```diff
@@ -1,15 +1,8 @@
 package com.codeforcommunity.rest;
 
-import com.codeforcommunity.api.IAuthProcessor;
-import com.codeforcommunity.api.IImportProcessor;
-import com.codeforcommunity.api.IProtectedUserProcessor;
-import com.codeforcommunity.api.IReservationProcessor;
+import com.codeforcommunity.api.*;
 import com.codeforcommunity.auth.JWTAuthorizer;
-import com.codeforcommunity.rest.subrouter.AuthRouter;
-import com.codeforcommunity.rest.subrouter.CommonRouter;
-import com.codeforcommunity.rest.subrouter.ImportRouter;
-import com.codeforcommunity.rest.subrouter.ProtectedUserRouter;
-import com.codeforcommunity.rest.subrouter.ReservationRouter;
+import com.codeforcommunity.rest.subrouter.*;
```

Comment:
Looks like your intellij is still auto importing with a star, can you change that setting so it stops doing this? Just search import in preferences and it should be under Java code style

Reply:
Can't turn it off but set it to do this at 20 imports which I doubt I'll ever exceed.